client/src/components/NetworkDashboard.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+const NetworkDashboard = () => {
+    //Obtain Available Network Information from User's Browser
+    const networkInterface = navigator.connection 
+                             || navigator.mozConnection 
+                             || navigator.webkitConnection;
+
+    //Creates an Array of Arrays of the Network Data for Mapping
+    let network = [];
+
+    if(networkInterface !== undefined) {
+        network = Object.entries({
+            type: networkInterface.type,
+            effectiveType: networkInterface.effectiveType,
+            downlink: networkInterface.downlink,
+            downlinkMax: networkInterface.downlinkMax,
+            rtt: networkInterface.rtt,
+            saveData: networkInterface.saveData,
+            onchange: networkInterface.onchange
+        });
+    }
+
+    //Network Information Key to Display for Users
+    const networkKey = {
```

Comment:
Totally support this idea from UI/UX view.  Making the keys more intuitive for user is a great idea.